students/AnastasiaPletser/hw-3/answers_1.md

```diff
@@ -0,0 +1,40 @@
+function makeSomeCoolStaff() {
```

Comment:
The moment when we declare a function is not the same to moment we run the function.
Please review your answers with that in mind.